src/routes/Review/components/ReviewTable/index.jsx

```diff
@@ -1,188 +1,39 @@
-import { Link } from "@reach/router";
-import classNames from "classnames";
-import PageDivider from "components/PageDivider";
-import { ProgressLevels } from "constants/";
-import React, { useState } from "react";
-import _ from "lodash";
-import ArrowIcon from "../../../../assets/images/icon-arrow.svg";
-import "./styles.module.scss";
+import React from "react";
+import { WorkType } from "../../../../../src-ts/lib";
+import { WorkDetailDetailsPane } from "../../../../../src-ts/tools/work/work-detail-details/work-detail-details-pane";
```

Comment:
we should not be importing components from within the work subfolders. we should export the details pane from the work-details and it will automatically be exported from the tools barrel file